pkg/broker/logic.go

```diff
@@ -39,6 +42,33 @@ func (b *BusinessLogic) GetCatalog(c *broker.RequestContext) (*broker.CatalogRes
 	return response, nil
 }
 
+func (b *BusinessLogic) ActionRotateCredentials(InstanceID string, vars map[string]string, context *broker.RequestContext) (interface{}, error) {
+	instance, err := b.GetInstanceById(InstanceID)
+	if err != nil {
+		return nil, NotFound()
+	}
+
+	provider, err := GetProviderByPlan(b.namePrefix, instance.Plan)
+	if err != nil {
+		glog.Errorf("Unable to rotate access keys, cannot find provider (GetProviderByPlan failed): %s\n", err.Error())
+		return nil, InternalServerError()
+	}
+
+	user, err := provider.RotateCredentials(instance)
+	if err != nil {
+		glog.Errorf("Unable to rotate access keys, RotateCredentials failed: %s\n", err.Error())
+		return nil, InternalServerError()
+	}
+
+	err = b.storage.UpdateCredentials(instance, user)
+	if err != nil {
+		glog.Errorf("Error: Unable to record password change for instance %s and user %s with new password %s\n", instance.Name, user.AccessKeyId, user.SecretAccessKey)
+		return nil, InternalServerError()
```

Comment:
Lets not write the password to logs, just knowing the name and access key id should be enough to fix the error, no?